apps/server/src/modules/lesson/service/lesson-copy.service.ts

```diff
@@ -227,7 +228,7 @@ export class LessonCopyService {
 				copiedContent.push(linkContent);
 				copiedContentStatus.push(embeddedTaskStatus);
 			}
-			if (element.component === ComponentType.NEXBOARD && nexboardEnabled) {
+			if (element.component === ComponentType.NEXBOARD && nexboardEnabled && copyNexboardEnabled) {
```

Comment:
the previous solution with status NOT_DOING was better
so, why not applying the condition just at that level instead of skiping the who thing?
the correct behaviour is to inform the client that nexboard is not copied (on purpose).